### ISSUE TITLE
Wix v4

### ISSUE THREAD

**Issue by Steve0212a** [flcdrg/PowerShellWixExtension#42]
I have tried to use your extension with the newly released Wix v4 (https://wixtoolset.org/docs/releasenotes/), but I am getting compile errors.

I upgraded a working v3 project, but when I try to compile the MSI in v4, I get the following error.
`error WIX0200: The Fragment element contains an unhandled extension element 'Script'. Please ensure that the extension for elements in the 'http://schemas.gardiner.net.au/PowerShellWixExtensionSchema' namespace has been provided.`

Here is my fragment (with a few things blanked out)
```
	<Fragment>
		<powershell:Script Id="xxx" Elevated="yes">
			<![CDATA[Get-ChildItem Cert:\LocalMachine\My | Where-Object {$_.Subject -match 'xxx' } | Remove-Item]]>
		</powershell:Script>

		...

	</Fragment>
```

Do you plan on supporting Wix v4?

thx

**Comment by flcdrg** [comment]
Yes, it would be good to support v4. I'd welcome any contributions towards this.

**Comment by Steve0212a** [comment]
I do not have the expertise, but hopefully someone can help.

thx
